### ISSUE TITLE
STAREPandas & pystare out of sync?

### ISSUE THREAD

**Issue by michaelleerilee** [SpatioTemporal/STAREPandas#66]
Looks like I need to update the stack.

```
path = geopandas.datasets.get_path('naturalearth_lowres')
world = geopandas.read_file(path)

roi = world[world.continent=='Oceania'] # Get a gdf from geopandas

# Convert the gdf to STARE
stare = starepandas.stare_from_gdf(roi, level=7, force_ccw=True)
roi = starepandas.STAREDataFrame(roi, stare=stare)

# The trixels are just for visualization
roi.set_trixels(inplace=True)
roi.head()
```

```---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/var/folders/q9/t6pps0j954l457p2dlg95bw00000gn/T/ipykernel_5622/4213342057.py in <module>
      9 
     10 # The trixels are just for visualization
---> 11 roi.set_trixels(inplace=True)
     12 roi.head()

~/opt/anaconda3/envs/stare/lib/python3.9/site-packages/starepandas/staredataframe.py in set_trixels(self, trixels, inplace)
    115     def set_trixels(self, trixels=None, inplace=False):
    116         if trixels is None:
--> 117             trixels = self.trixels()
    118         if inplace:
    119             frame = self

~/opt/anaconda3/envs/stare/lib/python3.9/site-packages/starepandas/staredataframe.py in trixels(self, stare_column)
    108         for index_values in self[stare_column]:
    109             # This is maybe not ideal if every row has single SID
--> 110             trixels = starepandas.to_trixels(index_values, as_multipolygon=True)
    111             trixels_series.append(trixels)
    112         return trixels_series

~/opt/anaconda3/envs/stare/lib/python3.9/site-packages/starepandas/tools/stare_conversions.py in to_trixels(sids, as_multipolygon)
     76         # This is not ideal, but when we read sidecars, we get unit64 and have to cast
     77         sids = sids.astype(numpy.int64)
---> 78     latv, lonv = pystare._to_vertices_latlon(sids)
     79     for i in range(len(latv)):
     80         latv[i] = pystare.shiftarg_lat(latv[i])

AttributeError: module 'pystare' has no attribute '_to_vertices_latlon'
```

**Comment by michaelleerilee** [comment]
Updated to current master.

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/var/folders/q9/t6pps0j954l457p2dlg95bw00000gn/T/ipykernel_6149/4213342057.py in <module>
      5 
      6 # Convert the gdf to STARE
----> 7 stare = starepandas.stare_from_gdf(roi, level=7, force_ccw=True)
      8 roi = starepandas.STAREDataFrame(roi, stare=stare)
      9 

AttributeError: module 'starepandas' has no attribute 'stare_from_gdf'
```

**Comment by michaelleerilee** [comment]
https://starepandas.readthedocs.io/en/latest/readme/README.html#id1

```
path = geopandas.datasets.get_path('naturalearth_lowres')
world = geopandas.read_file(path)
africa = world[world.continent=='Africa']
stare = starepandas.stare_from_gdf(africa , level=7, force_ccw=True)
africa  = starepandas.STAREDataFrame(africa , stare=stare)
```

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
/var/folders/q9/t6pps0j954l457p2dlg95bw00000gn/T/ipykernel_6149/1779448720.py in <module>
      2 world = geopandas.read_file(path)
      3 africa = world[world.continent=='Africa']
----> 4 stare = starepandas.stare_from_gdf(africa , level=7, force_ccw=True)
      5 africa  = starepandas.STAREDataFrame(africa , stare=stare)

AttributeError: module 'starepandas' has no attribute 'stare_from_gdf'
```

**Comment by michaelleerilee** [comment]
I wanted to generate a cover for Tasmania to help OPeNDAP with its demo, but stare_from_gdf seems broken at the moment.

**Comment by NiklasPhabian** [comment]
I'd think this has nothing to do with pystare. STAREPandas usage drastically changed starting version v0.5.0. In particular we added temporal functions. Therefore anything that was previously called `stare_from*` is now `sids_from*`.

I am just noticing that the readthedocs does not reflect this. I need to see what got clogged up there.

tl;dr. Replace
```stare_from_gdf``` with ```sids_from_gdf```

**Comment by NiklasPhabian** [comment]
ok. Readthedocs was pointing at the wrong branch. I configured it to build the docs for master.

**Comment by NiklasPhabian** [comment]
so it is reflected here now:
https://starepandas.readthedocs.io/en/master/docs/reference/spatial_conversions.html

**Comment by NiklasPhabian** [comment]
ooh; ok. 'latest' was pointing to a stale and merged branch. It is now pointing to master.
So this will get us to the correct documentation.
https://starepandas.readthedocs.io/en/latest/docs/reference/spatial_conversions.html
